### DISCUSSION TITLE
ChatGPT 師兄弟 Ellie AI 幫你寫出有禮貌的信

### DISCUSSION THREAD

**Discussion by 32844583** [1101-GS3073-C/Python-Community-GS3073-C#508] [General]
### Ellie AI可自動撰寫信件　模仿人類口吻
OpenAI創建的自然語言對話模型ChatGPT近日在全球流行，被廣泛應用於各種用途。近期，OpenAI又推出了一款新的機器人Ellie AI，專門解決滿溢的電子郵件。Ellie AI能夠自動寫信回覆，並且還能選擇期望的信件回覆口吻，如正式、不正式、感興趣等。Ellie AI於12月14日推出，已有超過500人試用。

https://www.bnext.com.tw/article/73301/ellie-ai-leave-me-alone-squarecat-20221219131606-kqf3ygnr

**Comment by manipulateed** [comment]
回覆別人的信件用自動寫信感覺不怎麼有誠意，但有些時候忘記或是沒時間寫吧，這也算是很好的幫手

**Comment by NcuJerryKao** [comment]
ChatGPT 最近好紅喔 怎麼到處都能看到 另一門通識課上看提到
在YT推薦中也可以看到

改天也依定要來玩玩看

**Comment by williamyen0623** [comment]
ChatGPT真的好神，用過都說讚，讚嘆科技的進步！

**Comment by BingRong1217** [comment]
現在的科技可以幫助人類完成好多事情

**Comment by xCk27x** [comment]
以後寫信跟老師要密碼卡就靠它了哈哈~

**Comment by limitx0** [comment]
每次寫信都不知道要寫什麼
能讓AI幫忙就太好了

**Comment by Chenszuhong** [comment]
這個主題真是不錯的想法 感覺都很有趣而且很帥 想要知道更多能延生應用的地方

**Comment by larry40305** [comment]
寄信給教授的時候非常需要這種AI

**Comment by boson13579** [comment]
Ellie AI最好用在學習如何寫信就好，畢竟寫信還是自己寫比較有誠意。

**Comment by boson13579** [comment]
Ellie AI最好用在學習如何寫信就好，畢竟寫信還是自己寫比較有誠意。
